examples/sql/client-api/employee/main.go

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+This example demonstrates how to use the Hazelcast database/sql driver.
```

Comment:
Is this showing sql driver usage? I think driver usage is like this:

```go
db := sql.Open("DSN")
db.ExecSQL("SELECT..")
```

I bet this is a leftover comment.

Reply:
That's right, copy/paste hazard. Removed at: https://github.com/hazelcast/hazelcast-go-client/pull/728/commits/858ee6604d144eb445053d128ef8764ce720e156